content/boilerplates/notes/1.2.1.md

```diff
@@ -0,0 +1,16 @@
+## Bug fixes and minor enhancements
+
+- Fix duplicate CRD being generated in the install ([Issue 14976](https://github.com/istio/istio/issues/14976))
+- Allow users to disable istio default retries by setting retries to 0 ([Issue 14900](https://github.com/istio/istio/issues/14900))
+- Improve citadel log ([Pull Request 15032](https://github.com/istio/istio/pull/15032))
+- Add citadel replicaCount flag back and workloadCertTtl to support multi-citadel and short cert TTL for perf test ([Pull Request 15066](https://github.com/istio/istio/pull/15066))
```

Comment:
Does it make sense to say allow users to run multiple replicas of citadel now via `citadel.replicaCount` (or something like that.)

We need more explanation on short cert TTL.  Why users need to pay attention to that?

Reply:
Maybe we should remove this from release notes.
I've mainly added those from PR, but it looks like a couple of PR are to improve Citadel debugging and logging, @wattli to confirm?

Reply:
```suggestion
```